app/presenters/project_presenter.rb

```diff
@@ -98,10 +105,10 @@ def as_json(_options = nil)
       gitRemote: git_remote,
       block: block,
       isFullyReleased: fully_released?,
-      isKubernetes: is_kubernetes?,
+      isKubernetes: kubernetes?,
```

Comment:
If possible, I'd prefer the client-side names to match the server-side ones (aside from casing).

Reply:
This one is weird -- `rubocop` says i need to use `kubernetes?` instead of `is_kubernetes?`, however it is JS convention to preface booleans with `isFoo`... open to changing but could also disable the rule here.

Reply:
We should definitely keep `kubernetes?` on the ruby side, where the convention is to use `?` names instead of `is`. Fine with me if you like the `is` convention on the JS side, as long as we're consistent.